### PR TITLE
Feature/route game play main with game mode

### PR DESCRIPTION
**Pull request by OniWarrior**
In this PR, the route from child dashboard to gameplay, and then, to gamemode component has been created.

Motivation.
We wanted the child dashboard to automatically go to the gamemode component after the accept the mission button was pressed by the child.

Changes.
1.In Gamemode component on line 59, I deleted a Header component. There were two header components being rendered and I simply removed this one so only one header component is being rendered.

2.In RenderChildDashboard on line 18, I changed the redirect from gameplay/mission/read to gameplay/gamemode. That way you're redirected to the gamemode component.

3.In GamePlayMain on line 49, I created the route to gamemode component.

Trello Card: https://trello.com/c/DmFHv7Sv/685-game-play-logic-routing-path-cleanup

PR submission video(Has a demo towards the end): https://drive.google.com/file/d/17uNqm5Pmp85yqfO1yukqU20L991g8SrE/view?usp=sharing

### CODE REVIEW

**Review by Yonathan-Admasu728 — DISMISSED** [review]
great fix in all 3 files.

File: src/components/pages/ChildDashboard/RenderChildDashboard.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const RenderChildDashboard = props => {
   const [modalVisible, setModalVisible] = useState(false);
 
   const handleAcceptMission = e => {
-    push('/gameplay/mission/read');
+    push('/gameplay/gamemode');
```
Comment (just-kato) [inline src/components/pages/ChildDashboard/RenderChildDashboard.js]: Nice route change

File: src/components/pages/Gamemode/Gamemode.js [hunk 2]
```diff
@@ -56,7 +56,6 @@ const Gamemode = ({ ...props }) => {
         : props.child.gamemode.mode === 'select' &&
           !props.child.gamemode.sp && (
             <div className="dash-container">
-              <Header />
```
Comment (just-kato) [inline src/components/pages/Gamemode/Gamemode.js:59]: Looked around and the Header tag was unnecessarily there

File: src/components/pages/GamePlay/GamePlayMain.js [hunk 3]
```diff
@@ -45,6 +46,9 @@ function GamePlayMain(props) {
             enableModalWindow={enableModalWindow}
           />
         </Route>
+        <Route path={`${baseURL}/gamemode`}>
+          <Gamemode baseURL={baseURL} />
```
Comment (just-kato) [inline src/components/pages/GamePlay/GamePlayMain.js:50]: Nice job with adding in the route!

**Review by just-kato — DISMISSED** [review]

**Review by just-kato — DISMISSED** [review]

File: src/components/pages/Gamemode/Gamemode.js (hunk 2 above)
Comment (artofmayhem) [reply src/components/pages/Gamemode/Gamemode.js:59]: Nice catch getting rid of that doubled header

File: src/components/pages/ChildDashboard/RenderChildDashboard.js (hunk 1 above)
Comment (artofmayhem) [reply src/components/pages/ChildDashboard/RenderChildDashboard.js]: gameplay/gamemode page will then point to gameplay/mission/read ad a sibling to gamemode. Nice work stitching that in there.

**Review by artofmayhem — APPROVED** [review]
Well executed and micely wired in. Ready for merge

**Review by ashtilawat — APPROVED** [review]

**Pull request merged**